src/server.rs

```diff
@@ -114,14 +114,12 @@ where
     async fn detach_node(&self, id: &Uuid) -> Resp<Empty> {
         info!("Node {id} Detach Request");
 
-        let mut nodes = self.nodes.write().await;
-
-        let removed = self.astarte_handler.unsubscribe(id).await?;
+        let removed = self.astarte_handler.unsubscribe(id, false).await?;
```

Comment:
Let's comment on this, so we make it clear that we will update it in the next version